### DISCUSSION TITLE
Wired controller

### DISCUSSION THREAD

**Discussion by type-rke** [JanM321/esphome-lg-controller#32] [General]
Hi, i'm looking for a solution for 3 wall and 2 floor units, is it possible with these to have local control?

now i have the wifi module but there is a lot of delay

**Comment by JanM321** [comment]
> Hi, i'm looking for a solution for 3 wall and 2 floor units, is it possible with these to have local control?

Yes, everything is local. The ESPHome code only communicates with Home Assistant.

That said, the wifi module supports more features but you can also use both at the same time.

**Reply by TerryFrench** [reply]
Also they likely came with some infra-red remote controls, and they still work.
